domain/host/host.go

```diff
@@ -168,11 +168,13 @@ func Build(ip string, rpcport string, poolid string, memory string, ipAddrs ...s
 	}
 	host, err := currentHost(ip, rpcPort, poolid)
 	if err != nil {
+		glog.Errorf("currentHost failed: %s", err)
```

Comment:
Could we do this one V(2)? We use this pattern throughout, but it (sometimes) leads to situations where we swallow or switch on an error at a higher level, but can't avoid the scary log message lower down.